test/integration/grpc_management_plane_api_test.go

```diff
@@ -177,16 +187,102 @@ func TestGrpc_ConfigUpload(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, http.StatusOK, resp.StatusCode())
 
-	client = resty.New()
+	responses := getManagementPlaneResponses(t, 2)
+
+	assert.Equal(t, mpi.CommandResponse_COMMAND_STATUS_OK, responses[0].GetCommandResponse().GetStatus())
```

Comment:
Why is this being checked twice ?

Reply:
The mock management plane is sending a config upload request on startup automatically. Thats why there are 2 responses